centml/cli/cluster.py

```diff
@@ -3,10 +3,23 @@
 import platform_api_client
 from platform_api_client.models.endpoint_ready_state import EndpointReadyState
 from platform_api_client.models.deployment_status import DeploymentStatus
-
+from .utils import client_certs
 from ..sdk import api
 
 
+# Custom class to parse key-value pairs for env variables for inference deployment
+class InferenceEnvType(click.ParamType):
+    name = "key_value"
+
+    def convert(self, value, param, ctx):
+        try:
+            key, val = value.split('=', 1)
+            return key, val
+        except ValueError:
+            self.fail(f"{value} is not a valid key=value pair", param, ctx)
+            return None  # to avoid warning from lint for inconsistent return statements
+
+
 hw_to_id_map = {"small": 1000, "medium": 1001, "large": 1002}
```

Comment:
Another ticket you can create. Currently the instance ids are hard-coded. We need to fetch this from the API.

Reply:
sure let me create a ticket for the same

Reply:
https://github.com/CentML/centml-python-client/issues/50